### DISCUSSION TITLE
Issues with Student Verification on GitHub Education due to Poor Camera Quality

### DISCUSSION THREAD

**Discussion by heeseon16** [community/community#133462] [GitHub Education]
### Hi everyone,

I'm trying to verify my student status on GitHub Education, but the image quality from my laptop and phone cameras is too poor, and the verification keeps failing. I noticed that other people have an 'upload image' button, but I don't have it. I've tried several times with my computer and phone cameras, but it's not working. What should I do?
![image](https://github.com/user-attachments/assets/7cbdaa3b-5658-40ed-974f-1e6568983e64)
Currently, I am at school, but GitHub is asking me to verify my location. What should I do?
![image](https://github.com/user-attachments/assets/b30622d6-f594-4af0-b7f9-0a6d81dd122c)

**Comment by adrianweidig** [comment]
Same here there should be an option or Mail where you can send your certificate of matriculation
